python-sdk/tests_integration/sql/operators/test_raw_sql.py

```diff
@@ -146,3 +147,57 @@ def assert_num_rows(results):
         assert_num_rows(results)
 
     test_utils.run_dag(sample_dag)
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [{"database": Database.SQLITE, "file": File(path=str(DATA_FILEPATH))}],
+    indirect=True,
+    ids=["sqlite"],
+)
+def test_run_raw_sql__results_format__pandas_dataframe(sample_dag, database_table_fixture):
+    """run_raw_sql() command should return `pandas.DataFrame` when `results_format='pandas_dataframe' is passed"""
+    _, test_table = database_table_fixture
+
+    @aql.run_raw_sql(results_format="pandas_dataframe")
+    def raw_sql_query(input_table):
+        return "SELECT * from {{input_table}}"
+
+    @task
+    def assert_num_rows(result):
+        assert isinstance(result, pandas.DataFrame)
+        assert result.shape == (1, 2)
+
+    with sample_dag:
+        results = raw_sql_query(input_table=test_table, response_size=1)
+        assert_num_rows(results)
+
+    test_utils.run_dag(sample_dag)
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [{"database": Database.SQLITE, "file": File(path=str(DATA_FILEPATH))}],
+    indirect=True,
+    ids=["sqlite"],
+)
+def test_run_raw_sql__results_format__list(sample_dag, database_table_fixture):
+    """run_raw_sql() command should return `pandas.DataFrame` when `results_format='list' is passed"""
+    _, test_table = database_table_fixture
+
+    @aql.run_raw_sql(results_format="list")
+    def raw_sql_query(input_table):
+        return "SELECT name from {{input_table}}"
+
+    @task
+    def assert_num_rows(result):
+        assert isinstance(result, list)
+        assert len(result) == 3
```

Comment:
Can you confirm the actual result please

Reply:
Sure, I'll update the testcase.

Reply:
Done.